lib/diagnostic.js

```diff
@@ -13,12 +13,28 @@
 
 'use strict';
 
-const fs = require('fs'); // jshint ignore: line
+let fs = require('fs'); // jshint ignore: line
 
-const sumLines = (filename, callback) => {
+let fileName = process.argv[2];
+
+const sumLines = fs.readFile(fileName, (err, data) => {
   /* your response here */
-  callback();
-};
+  if(err){
+    console.log(err+': File failed to load. Please check file path.');
+  }
+  let lines = data.toString().split('\n');
```

Comment:
Your code fails on this line because `data` is undefined. That's because your function, sumLines, get no arguments passed in.